aten/src/ATen/native/Math.h

```diff
@@ -279,13 +279,13 @@ static inline float trigamma(float x) {
 static inline double calc_digamma(double x) {
   static double PSI_10 = 2.25175258906672110764;
   if (x == 0) {
-    return INFINITY;
+    return std::copysign(INFINITY, -x);
```

Comment:
Add a comment here and for the use of copysign below explaining why this happens